content/applications/inventory_and_mrp/manufacturing/subcontracting/resupply_subcontracting_lead_times.rst

```diff
@@ -0,0 +1,161 @@
+components, in order to manufacture the product and deliver it on time.
+
+.. important::
+   Like all lead times in Odoo, lead times for subcontracted products are only an estimate, and are
+   based on how long actions are *expected* to take.
+
+   Unforeseen circumstances can impact the completion of these actions, which means that lead times
+   should not be viewed as guarantees.
+
+Configuration
+=============
+
+When using the :doc:`Resupply Subcontractor on Order <subcontracting_resupply>` route, a company is
+responsible for supplying the subcontractor with the necessary components. As a result, the
+subcontractor cannot begin manufacturing until the components have been received.
+
+This means that, in addition to the amount of time it takes the subcontractor to manufacture and
+deliver the product, the date they receive the components must also be considered.
+
+By assigning a product's subcontractor a delivery lead time, and specifying a manufacturing lead
+time on the product's bill of materials (BoM), *Resupply Subcontractor* orders for the product's
+components display the deadline by which the subcontractor must receive the components.
+
+Product delivery lead time
+--------------------------
+
+To set a delivery lead time for a product's subcontractor, navigate to :guilabel:`Inventory app -->
+Products --> Products`, and select a subcontracted product.
+
+Then, select the :guilabel:`Purchase` tab on the product's page. If the subcontractor has not
+already been added as a vendor, do so now by clicking :guilabel:`Add a line`, and selecting the
+subcontractor in the :guilabel:`Vendor` field.
+
+Once the subcontractor has been added, enter the number of days it takes them to manufacture and
+deliver the product, in the :guilabel:`Delivery Lead Time` column.
+
+.. image:: resupply_subcontracting_lead_times/delivery-lead-time.png
+   :align: center
+   :alt: The Delivery Lead Time field for a subcontractor, on the Purchase tab of a product page.
+
+Product manufacturing lead time
+-------------------------------
+
+Next, navigate to the product's |BoM| by clicking the :guilabel:`Bill of Materials` smart button at
+the top of the product's page. Then, select a BoM from the list.
+
+On the |BoM|, select the :guilabel:`Miscellaneous` tab. In the :guilabel:`Manuf. Lead Time` field,
+enter the same number of days that was entered in the :guilabel:`Delivery Lead Time` field of the
+|BoM|'s product.
+
+.. image:: resupply_subcontracting_lead_times/manufacturing-lead-time.png
+   :align: center
+   :alt: The Manuf. Lead Time field on a product's BoM.
+
+While not all of these days will actually be used for manufacturing by the subcontractor, setting
+the same number of days in each field tells Odoo that the subcontractor must receive the components
+and begin production by the start of the product's delivery lead time. This gives the subcontractor
+enough time to both manufacture and deliver the product.
```

Comment:
Optional suggestion to put this info in a note or tip admonition because it seems good to remember